### ISSUE TITLE
Please make "Explore" breadcrumb a link to new explore page

### ISSUE THREAD

**Issue by nicholastmosher** [grafana/grafana#79837]
**Why is this needed**:

Often when using the explore page, I want to open a new blank explore page to test a new query on a blank page. I naturally want to `Cmd+Click` the Explore breadcrumb, but it's not a link.

<img width="279" alt="image" src="https://github.com/grafana/grafana/assets/4210949/86a72c1c-2de1-4b2c-8b7a-8f19ed0a6a34">

Compare this to the "Dashboards" breadcrumb when viewing a specific dashboard

<img width="214" alt="image" src="https://github.com/grafana/grafana/assets/4210949/ee898a02-531d-49ce-9139-ee5aade9b6f3">

I often `Cmd+Click` this to find and open another dashboard page. I noticed that on the "Dashboards" page itself (not a specific dashboard), the breadcrumb is also not clickable, and it feels like it would just be much nicer if it was for consistency

<img width="249" alt="image" src="https://github.com/grafana/grafana/assets/4210949/8271da96-3d08-4692-89ea-cca0d574d971">

**What would you like to be added**:

Please make the last segment of the breadcrumb trail a clickable link like the rest of the segments are. It would save me a ton of clicks navigating Grafana!

**Who is this feature for?**

Me, somebody who tries to have way too many tabs open at once

Thanks!

**Comment by gelicia** [comment]
Also look into adding a name for explore that would show up in the tab/title, and have that appear in the breadcrumb as a child to Explore

**Comment by gelicia** [comment]
We chatted about this on standup and want to have some further internal discussions with product/UX before moving forward. @diegoadams @thanos-karachalios 

A couple things to get defined by UX: For one, should this behavior be part of the core platform logic? Shouldn't this be standardized across all of Grafana? IE, in the dashboards root, would we want clicking on "dashboards" to refresh the dashboards page? https://play.grafana.org/dashboards 

Secondly, we thought about merging this with another idea to help make it consistent. @Elfo404 brought up wanting the ability to optionally name an explore instance so that title could show up as the tab name. Right now, the tab for Explore is simply the datasource name
![Screenshot 2024-01-02 at 10 32 21 AM](https://github.com/grafana/grafana/assets/1533128/20dee181-d1d5-4851-a8f0-93673e4b18cf)
If we add the ability to name Explore instances, we could add that name to the end of the breadcrumb. If you name an explore instance "Logs investigation", it would show up in the tab next to the Grafana logo, and the breadcrumb for this would be "Home > Explore > Logs investigation". Logs investigation wouldn't be clickable, but clicking Explore would prompt for losing changes (if there were some) and would open a blank explore instance if confirmed. Most importantly, it could be right clicked or cmd+clicked to open a new tab with a blank explore instance.

**Comment by diegoadams** [comment]
Regarding naming Explore instances, if a user chooses not to name it, would it remain as is like _Home > Explore._

In my opinion, breadcrumbs are meant exclusively for guiding users through their navigation path. By standard UX practices, the last  breadcrumb element that signifies the user's current location should not have interactive capabilities.

I would rather suggest maybe having a "New Explore" in the "Add" actions space
<img width="204" alt="Screenshot 2024-01-11 at 20 02 09" src="https://github.com/grafana/grafana/assets/95694434/9883c3df-1344-45b3-b30e-7e8a8f0d5a7a">

**Comment by Elfo404** [comment]
@diegoadams no, the idea is that the breadcrumbs would become "Home > Explore > Datasource name" or  "Home > Explore > Datasource1 name - Datasource2 name" in split view (mirroring the window title).

Users would then be able to rename "Datasource1 name - Datasource2 name" to whatever they want (i.e. "Logs investigation") (which again, would be mirrored in the window title) so that it would be easier to find when opening multiple tabs.

> I would rather suggest maybe having a "New Explore" in the "Add" actions space

that'd be pretty much pointless, users can already open a new explore tab by ctrl+clicking on the navigation menu, we simply want to make this a one click process instead of 2.
Additionally, the "Add" section is meant to add the content  of one explore pane to something else (i.e alert, dashboard), having an action there to "open a new window" would be quite inconsistent imo

**Comment by nicholastmosher** [comment]
> In my opinion, breadcrumbs are meant exclusively for guiding users through their navigation path. By standard UX practices, the last breadcrumb element that signifies the user's current location should not have interactive capabilities.

I'd just like to raise a counterpoint, in GitHub the last breadcrumb is clickable, and I regularly use this to Cmd+Click and open duplicate tabs on the same repository

<img width="204" alt="image" src="https://github.com/grafana/grafana/assets/4210949/c98d9037-069c-46e5-b22f-349d67bd1cf5">

> we simply want to make this a one click process instead of 2

💯 this is my main hope as well, it bugs me to need to open the side menu to get an explore link to Cmd+Click

**Comment by diegoadams** [comment]
@nicholastmosher , I have also consulted with the UX team on this view. 

However, if we go with the solution explained by @Elfo404 `Home > Explore > Datasource name` then yes it would make more sense to do the Cmd+Click on the `Explore` as it would be a level down. 
<img width="542" alt="Screenshot 2024-01-23 at 18 22 50" src="https://github.com/grafana/grafana/assets/95694434/b7c0b11c-c813-4afc-a435-7138814b5462">
